src/solver/type_getters.jl

```diff
@@ -80,23 +80,38 @@ function get_atmos(config::AtmosConfig, params)
         rayleigh_sponge = get_rayleigh_sponge_model(parsed_args, params, FT),
         sfc_temperature = get_sfc_temperature_form(parsed_args),
         surface_model = get_surface_model(parsed_args),
+        numerics = get_numerics(parsed_args),
     )
 
     @info "AtmosModel: \n$(summary(atmos))"
     return atmos
 end
 
 function get_numerics(parsed_args)
+    test_dycore =
+        parsed_args["test_dycore_consistency"] ? TestDycoreConsistency() :
+        nothing
+
+    energy_upwinding = Val(Symbol(parsed_args["energy_upwinding"]))
+    tracer_upwinding = Val(Symbol(parsed_args["tracer_upwinding"]))
+    density_upwinding = Val(Symbol(parsed_args["density_upwinding"]))
+    edmfx_upwinding = Val(Symbol(parsed_args["edmfx_upwinding"]))
+
+    limiter =
+        parsed_args["apply_limiter"] ? Limiters.QuasiMonotoneLimiter : nothing
```

Comment:
By the way, @Sbozzolo, this is probably the reason for the increased allocations. `Limiters.QuasiMonotoneLimiter` is a UnionAll type, and the compiler can struggle with inference on these types.